### PR TITLE
docs(refactor): updated H1 MTA-2012

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
Updated H1

### CODE REVIEW

File: compute/containers/api-cli/index.mdx [hunk 1]
```diff
@@ -3,6 +3,6 @@ meta:
   title: Serverless Containers - API/CLI Documentation
   description: Containers API/CLI Documentation
 content:
-  h1: API/CLI Documentation
+  h1: Serverless Containers - API/CLI Documentation
```
Comment (RoRoJ) [inline compute/containers/api-cli/index.mdx:6]: I think this should just be "Containers", to be ISO with the menu side bar. Serverless Containers is OK for the metadata though. Same for all the other affected H1s; and for Functions.

File: console/my-account/concepts.mdx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 meta:
-  title: Scaleway account - Concepts
+  title: Scaleway Account - Concepts
   description: This page explains all the concepts related to accounts
 content:
-  h1: Concepts
+  h1: Scaleway Account - Concepts
```
Comment (RoRoJ) [inline console/my-account/concepts.mdx:6]: Think this (and others) should be "My Account" like the side bar

File: console/my-project/api-cli/index.mdx [hunk 3]
```diff
@@ -3,6 +3,6 @@ meta:
   title: Projects - API/CLI Documentation
   description: My Project API/CLI Documentation
 content:
-  h1: API/CLI Documentation
+  h1: Projects - API/CLI Documentation
```
Comment (RoRoJ) [inline console/my-project/api-cli/index.mdx:6]: My Project (like the sidebar)

File: dedibox-network/network/troubleshooting.mdx [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
 meta:
-  title: Dedibox network - Troubleshooting
+  title: Dedibox Network - Troubleshooting
   description: This page explains techniques for how to troubleshoot network related issues on your Dedibox dedicated server. 
 content:
-  h1: Troubleshooting
+  h1: Dedibox Network - Troubleshooting
   paragraph: This page explains techniques how to troubleshoot network related issues on your Dedibox dedicated server.
```
Comment (nedaehm) [inline dedibox-network/network/troubleshooting.mdx:7]:
```suggestion
  paragraph: This page explains techniques how to troubleshoot network-related issues on your Dedibox dedicated server.
```

File: compute/functions/quickstart.mdx [hunk 5]
```diff
@@ -1,9 +1,9 @@
 ---
 meta:
-  title: Serverless Functions Quickstart
+  title: Functions - Quickstart
   description: This page shows you how to get started with Scaleway Serverless Functions
 content:
-  h1: Quickstart
+  h1: Sreverkess Functions - Quickstart
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply compute/functions/quickstart.mdx:6]:
```suggestion
  h1: Functions - Quickstart
```

[the remark this answers is not in the document]
Comment (bene2k1) [reply compute/functions/quickstart.mdx:6]:
```suggestion
  h1: Functions - Quickstart
```

File: compute/functions/index.mdx [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 meta:
-  title: Functions Documentation
+  title: ServerlessFunctions Documentation
```
Comment (RoRoJ) [inline compute/functions/index.mdx:3]:
```suggestion
  title: Functions Documentation
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**